code/src/components/Form.js

```diff
@@ -0,0 +1,78 @@
+import React, { useState, useEffect } from 'react'
+
+import Recent from './Recent'
+import Create from './Create'
+
+const Form = () => {
+
+  const [newThought, setnewThought] = useState('');
+  const [thoughts, setThoughts] = useState([]);
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    fetchThoughts();
+  }, [])
+
+  const fetchThoughts = () => {
+    setLoading(true)
+    fetch('https://happy-thoughts-technigo.herokuapp.com/thoughts')
+      .then(res => res.json())
+      .then(json => setThoughts(json))
+      .catch(error => console.error(error))
```

Comment:
Hm, what does this do? 
Did I miss something in class or is it a console log that shouldnt be here?

Reply:
Hmm, thats a really good question! I´m not sure, started with some error messages but didn´t continue, this might a rest from that, haha:)  I´ll check that!